src/screens/CreatePost/index.js

```diff
@@ -717,7 +715,7 @@ const CreatePost = () => {
       animatedReminder.setValue(0);
     }
   }, [typeUser]);
-  console.log({typeUser}, 'heja');
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The console.log statement has been removed. This is a good practice for production code as it improves performance and security by not logging potentially sensitive data.
```diff
-  console.log({typeUser}, 'heja');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->